stylecomponents/buttons.css

```diff
@@ -0,0 +1,20 @@
+button {
+  border: none;
+  background: white;
+}
+
+.card__button {
+  height: 2rem;
+  border: 2px solid var(--primarycolor);
+  margin: 1rem;
+}
+
+.button__profile {
+  border: 1px solid var(--primarycolor);
+  border-radius: 5px;
+  padding: 0.5rem;
+  text-align: center;
+}
+
+.button__profile--show {
+}
```

Comment:
Deciding whether a button is a custom component or not is not easy. In your case, looking at the associated stylings, it looks to me like you have basic button stylings (which can also be set with a class selector and not with the type selector).
`.card__button`, `.button__profile`, and `.button__profile-show` seem to me to be variants of this button. Therefore I would work here with the modifier: `.button-card` `.button-profile` and `.button-profile-active`. Since `.button__profile-show` has no CSS declarations, maybe you don't need this rule?